lib/task-data/schemas/install-os-general.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "rackhd/schemas/v1/task-schema",
```

Comment:
Actually, I fall into trouble about how to find a good schema name. I make a little complex structure to accommodate the slight difference between different install-os tasks and for sake of schema reuse:

- `install-os-base` is the basic install-os schema, with minimum parameters properties and includes all parameters' definitions, though some of the definition will not be used by install-os-base itself.

- `install-os-general` is the general parameters for most install-os tasks (Ubuntu/SUSE/CentOS/Redhat/ESXi), CoreOS is an exception because it only supports very limited parameters.  `install-os-general` extends from `install-os-base`.

- `install-centos`/`install-exsi`: These schema extends from `install-os-general`, to define the schema for these particular OS as they have some very special parameters support, such as the `rachdCallbackScript` for ESXi/CentOS and the `postInstallCommands`/`switchDevices` for ESXi.

- `install-coreos`: The special schema for CoreOS, as it is very different from other OS.